ppr-ui/src/components/common/DischargeConfirmSummary.vue

```diff
@@ -0,0 +1,128 @@
+          <p class="ma-0 pt-2"><b>Registration Type:</b> {{ regType }}</p>
+          <p class="ma-0 pt-2"><b>Collateral:</b> {{ collateralSummary }}</p>
+        </div>
+        <v-checkbox
+          id="discharge-confirm-checkbox-1"
+          class="ma-0 pt-4"
+          :class="showErrors && !checkbox1 ? 'check-box-error': 'copy-normal'"
+          hide-details
+          v-model="checkbox1"
+        >
+          <template v-slot:label>
+            <p class="ma-0">
+              I confirm that I wish to <b>discharge this registration.</b>
+            </p>
+          </template>
+        </v-checkbox>
+        <v-checkbox
+          id="discharge-confirm-checkbox-2"
+          class="ma-0 pt-4"
+          :class="showErrors && !checkbox2 ? 'check-box-error': 'copy-normal'"
+          hide-details
+          v-model="checkbox2"
+        >
+          <template v-slot:label>
+            <p class="ma-0">
+              I understand that <b>all collateral on this registration will be released.</b>
+            </p>
+          </template>
+        </v-checkbox>
+        <v-checkbox
+          id="discharge-confirm-checkbox-3"
+          class="ma-0 pt-4"
+          :class="showErrors && !checkbox3 ? 'check-box-error': 'copy-normal'"
+          hide-details
+          v-model="checkbox3"
+        >
+          <template v-slot:label>
+            <p class="ma-0">
+              I understand that <b>all Secured Parties will be notified.</b>
+            </p>
+          </template>
+        </v-checkbox>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  reactive,
+  toRefs,
+  watch
+} from '@vue/composition-api'
+
+// local
+import { UIRegistrationTypes } from '@/enums' // eslint-disable-line no-unused-vars
+
+export default defineComponent({
+  name: 'DischargeConfirmSummary',
+  props: {
+    setRegNum: String,
+    setRegType: String,
+    setCollateralSummary: String,
+    setShowErrors: {
+      default: false
+    }
+  },
+  emits: ['valid'],
+  setup (props, { emit }) {
+    const localState = reactive({
+      checkbox1: false,
+      checkbox2: false,
+      checkbox3: false,
+      collateralSummary: props.setCollateralSummary,
+      regNum: props.setRegNum,
+      regType: props.setRegType as UIRegistrationTypes,
+      showErrors: props.setShowErrors,
+      valid: computed(() => {
+        if (localState.checkbox1 && localState.checkbox2 && localState.checkbox3) {
+          return true
+        }
+        return false
+      })
+    })
+
+    watch(() => props.setShowErrors, (val) => {
+      localState.showErrors = val
+    })
+    watch(() => localState.valid, (val) => {
+      emit('valid', val)
+    })
+
+    return {
+      ...toRefs(localState)
+    }
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+// .check-box-error ::v-deep .v-icon.notranslate.mdi.mdi-checkbox-blank-outline.theme--light,
```

Comment:
remove?

Reply:
Yeah sure, I was thinking I'd keep that in case they want the checkboxes to turn red as well as the words, but I'll just save that somewhere